docs-next/src/content/docs/running/cli.mdx

```diff
@@ -0,0 +1,625 @@
+---
+title: Command-Line Usage
+description: Running Mocha in your terminal.
+---
+
+Running `npx mocha --help` will show the full list of available commands:
```

Comment:
Should we add a source comment linking this to the definition in code? I want to update some stuff here but want it to match released code too

Reply:
Good candidate for a followup PR